src/test/java/uk/gov/hmcts/reform/sendletter/services/ReportsServiceTest.java

```diff
@@ -0,0 +1,151 @@
+            .containsExactlyInAnyOrder(
+                new LettersCountSummary("ServiceA", 10),
+                new LettersCountSummary("ServiceB", 20));
+    }
+
+    @Test
+    void should_return_letters_count_excluding_the_test_service() {
+        LocalDate date = LocalDate.of(2019, 4, 25);
+        LocalTime timeFrom = LocalTime.parse("17:00");
+        LocalTime timeTo = LocalTime.parse("16:00");
+
+        //given
+        given(repository.countByDate(formatDateTime(date.minusDays(1), timeFrom), formatDateTime(date, timeTo)))
+            .willReturn(asList(
+                new ServiceLettersCount("aService", 10),
+                new ServiceLettersCount("send_letter_tests", 20)
+            ));
+        given(serviceFolderMapping.getFolderFor("aService")).willReturn(Optional.of("ServiceA"));
+        given(serviceFolderMapping.getFolderFor("send_letter_tests")).willReturn(Optional.of("BULKPRINT"));
+
+        //when
+        List<LettersCountSummary> result = service.getCountFor(date);
+
+        //then
+        assertThat(result).isNotEmpty()
+            .hasSize(1)
+            .usingFieldByFieldElementComparator()
+            .containsExactlyInAnyOrder(new LettersCountSummary("ServiceA", 10));
+    }
+
+    @Test
+    void should_return_letters_count_excluding_the_nulls() {
+        LocalDate date = LocalDate.of(2019, 4, 25);
+        LocalTime timeFrom = LocalTime.parse("17:00");
+        LocalTime timeTo = LocalTime.parse("16:00");
+
+        //given
+        given(repository.countByDate(formatDateTime(date.minusDays(1), timeFrom), formatDateTime(date, timeTo)))
+            .willReturn(asList(
+                new ServiceLettersCount("aService", 10),
+                new ServiceLettersCount(null, 2)
+            ));
+        given(serviceFolderMapping.getFolderFor("aService")).willReturn(Optional.of("ServiceA"));
+
+        //when
+        List<LettersCountSummary> result = service.getCountFor(date);
+
+        //then
+        assertThat(result).isNotEmpty()
+            .hasSize(1)
+            .usingFieldByFieldElementComparator()
+            .containsExactlyInAnyOrder(new LettersCountSummary("ServiceA", 10));
+    }
+
+    @Test
+    void should_map_empty_list_from_repo() {
+        LocalDate date = LocalDate.of(2019, 4, 25);
+        LocalTime timeFrom = LocalTime.parse("17:00");
+        LocalTime timeTo = LocalTime.parse("16:00");
+
+        //given
+        given(repository.countByDate(formatDateTime(date.minusDays(1), timeFrom), formatDateTime(date, timeTo)))
+            .willReturn(Collections.emptyList());
+
+        //when
+        List<LettersCountSummary> result = service.getCountFor(date);
+
+        //then
+        assertThat(result).isEmpty();
+    }
+
+    private LocalDateTime formatDateTime(LocalDate date, LocalTime time) {
+        ZonedDateTime zonedDateTime = ZonedDateTime.of(date, time, ZoneId.from(ZoneOffset.UTC));
+        return zonedDateTime.toLocalDateTime();
+    }
```

Comment:
I think this is second usage of same implementation. Perhaps worth moving to `uk.gov.hmcts.reform.sendletter.util.TimeZones` utility class? Happy for it to go in another PR

Reply:
moved it to TimeZones utility class